examples/suspense/src/use_sleep.rs

```diff
@@ -0,0 +1,43 @@
+use std::rc::Rc;
+
+use gloo_timers::future::TimeoutFuture;
+use wasm_bindgen_futures::spawn_local;
+use yew::prelude::*;
+use yew::suspense::{Suspension, SuspensionResult};
+
+#[derive(PartialEq)]
+pub struct SleepState {
+    s: Suspension,
+}
+
+impl SleepState {
+    fn new() -> Self {
+        let (s, handle) = Suspension::new();
+
+        spawn_local(async move {
+            TimeoutFuture::new(5_000).await;
```

Comment:
Why not use [`sleep`](https://docs.rs/gloo-timers/latest/gloo_timers/future/fn.sleep.html)?